kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -113,6 +113,7 @@
       const store = getCurrentInstance().proxy.$store;
       const route = computed(() => store.state.route);
       const topicId = computed(() => route.value.params.topic_id);
+      const bookMarksFlag = computed(() => route.value.query.bookmarks);
```

Comment:
Blocking: I think that maybe we should have the query parameter & the name of this computed property be the same - so perhaps to be clear about what happens when this is true, it could be something like `showBookmarks` - and the query param the same `?showBookmarks`.